### DISCUSSION TITLE
Integrate with SuperTokens

### DISCUSSION THREAD

**Discussion by yt-dev** [refinedev/refine#4233] [Q&A] [answered]
🤔 How to integrate SuperTokens into a Refine app that uses the next.js framework.

**Comment by omeraplak** [comment, marked as answer]
Hey @yt-dev ,
Thank you for contacting us. While integrating SuperTokens, you can review the examples available here: https://refine.dev/docs/api-reference/core/providers/auth-provider/#examples

https://refine.dev/docs/tutorial/understanding-authprovider/index/

**Reply by yt-dev** [reply]
Oops, thanks.
